core/modules/file.py

```diff
@@ -21,11 +21,11 @@
 from viur.core.prototypes.tree import SkelType, Tree, TreeSkel
 from viur.core.skeleton import skeletonByKind
 from viur.core.tasks import PeriodicTask, CallDeferred
-from viur.core.utils import projectID, sanitizeFileName
+from viur.core.utils import  sanitizeFileName
 
 credentials, project = google.auth.default()
 client = storage.Client(project, credentials)
-bucket = client.lookup_bucket("%s.appspot.com" % projectID)
+bucket = client.lookup_bucket("%s.appspot.com" % conf["viur.instance.project_id"])
```

Comment:
I love f-strings ;-)
```suggestion
bucket = client.lookup_bucket(f"""{conf["viur.instance.project_id"]}.appspot.com""")
```